### PR TITLE
Correct the extauth grpc passthroughauth retryproto comment

### PR DESCRIPTION
**Pull request by davidjumani**
# Description

Correct the extauth grpc passthroughauth retryproto comment.
It incorrectly mentions that a default retry will be configured when that is the intended behaviour

## API changes
- N/A

## Code changes
- N/A

## CI changes
- N/A

## Docs changes
- N/A

# Context
- N/A

# Checklist:

- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: projects/gloo/api/v1/enterprise/options/extauth/v1/extauth.proto [hunk 1]
```diff
@@ -1389,9 +1389,8 @@ message PassThroughGrpc {
     // TLS config for the Grpc passthrough, if not configured the connection will use insecure.
     PassThroughGrpcTLSConfig tlsConfig =3;
 
-    // Indicates the retry policy for re-establishing the gRPC stream
-    // This field is optional. If max interval is not provided, it will be set to ten times the provided base interval.
-    // If not set, gRPC streams default base interval:500ms, maximum interval:30s will be applied.
+    // Indicates the retry policy for re-establishing the gRPC stream.
+    // This field is optional and failed calls will not not retry unless configured.
```
Comment (sheidkamp) [inline projects/gloo/api/v1/enterprise/options/extauth/v1/extauth.proto:1393]: The extauth code works this way if retry policy is nil, but are we always setting a retry policy here: https://github.com/solo-io/solo-projects/blob/c434d34dffc641b87e47458781b013f8e900ae6a/projects/extauth/pkg/config/translator.go#L745 ?

**Review by sheidkamp — COMMENTED** [review]
one question

File: projects/gloo/api/v1/enterprise/options/extauth/v1/extauth.proto (hunk 1 above)
Comment (nfuden) [reply projects/gloo/api/v1/enterprise/options/extauth/v1/extauth.proto:1393]: Are we also setting it always in the kube controller variant?

Comment (sheidkamp) [reply projects/gloo/api/v1/enterprise/options/extauth/v1/extauth.proto:1393]:
```suggestion
    // This field is optional and failed calls will not retry unless configured.
```

Comment (sheidkamp) [reply projects/gloo/api/v1/enterprise/options/extauth/v1/extauth.proto]: @nfuden - we are setting it here: https://github.com/solo-io/ext-auth-service/blob/32bee3462fc556c40a2c9681a3ada7b70d08ffb3/pkg/controller/translation/pass_through.go#L89 

but in that case it should be nil, and the Retry opts are not added:
 https://github.com/solo-io/ext-auth-service/blob/32bee3462fc556c40a2c9681a3ada7b70d08ffb3/pkg/config/passthrough/grpc/grpc_client_manager.go#L73

**Review by sheidkamp — APPROVED** [review]
lgtm

**Review by jbohanon — APPROVED** [review]

**Pull request merged**